.github/pull_request_template.md

```diff
@@ -3,6 +3,8 @@
 ## What
 <!-- Describe your changes here -->
 
+<!-- If you are doing a front end change, please include a screen recording -->
```

Comment:
we could also mention the #feature-recordings slack channel in this comment and mention the use of possible screen recording tools that Foragers use (like Loom).

Reply:
I'll just mention the slack channel. Don't want to mention Loom yet because not sure if it will become heavily adopted